documentation/modules/proc-logging-in-console.adoc

```diff
@@ -6,7 +6,21 @@
 = Accessing the {ConsoleName}
 
 .Prerequisites
-* You must have obtained the host name for the {ConsoleName}. For more information about how to obtain the host name, see  link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#retrieving-address-space-information-messaging[Example commands for retrieving address space information].
+ifeval::["{cmdcli}" == "oc"]
+* On OpenShift Container Platform 3.x, obtain the host name for the {ConsoleName} by running the following command:
+----
+{cmdcli} get routes console -o jsonpath={.spec.host}
+----
+
+* On OpenShift Container Platform 4.x, obtain the host name for the {ConsoleName} by running the following command:
+----
+oc get consolelink -l app=enmasse -o jsonpath={.spec.href}
+----
+endif::[]
+
+ifeval::["{cmdcli}" == "kubectl"]
+* ??
```

Comment:
@k-wall Just occurred to me that we also need to include the command for Kubernetes access as well, correct?